app/helpers/application_helper.rb

```diff
@@ -1,2 +1,28 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  def show_errors object, field_name
+    return unless object.errors.any? &&
+                  object.errors.messages[field_name].present?
+
+    object.errors.messages[field_name].join(", ")
+  end
+
+  def time_ago_in_words_custom time
```

Comment:
dùng hàm có sẵn của rails dc k e?
https://stackoverflow.com/questions/28667064/how-to-use-rails-datehelper-method-time-ago-in-words-outside-of-rails

Reply:
Em thêm 1 file như này nhé
https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/vi.yml